desktop/src/main/java/com/group5/java/EndlessRunnerDesktop.java

```diff
@@ -14,6 +14,7 @@
      */
     public static void main(final String[] args) {
         LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
+        config.title = "\uD83D\uDC14";
```

Comment:
This is a chicken emoticon, please add as a named variable or put comment to make this clear to possible code reader.